errbot/errBot.py

```diff
@@ -417,10 +424,12 @@ def _process_command(self, mess, cmd, args, match):
                                    template_name=f._err_command_template)
 
     @staticmethod
-    def process_template(template_name, template_parameters):
+    def process_template(template_name, template_parameters, plugin_name):
```

Comment:
you'd move this part into plugin instead of bot? Then it would be possible to call it on `f.__self__` below.
Then the question is if the method here (or rather, send_templated) would be needed still.